rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.77.2"
+channel = "1.79.0"
```

Comment:
This is fine, but normally handled in operator-templating (I'd forgotten). See https://github.com/stackabletech/operator-templating/pull/406

Reply:
You don't need to change anything, the templating run will just skip over that change.